cpp/collections_module/algorithm/algorithm.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <mgp.hpp>
+
+#include <exception>
+#include <list>
+#include <stdexcept>
+
+namespace Collections{
+
+constexpr const char *kResultAverage = "average";
+
+void Avg(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory);
+
+}
```

Comment:
This red circle on GitHub means that you are missing an empty line at the end of the file. Practice is on our side to always have an empty line after the last line in the file.